### DISCUSSION TITLE
Using Juggluco, sensor damage issue.

### DISCUSSION THREAD

**Discussion by sidouzi123** [j-kaltes/Juggluco#141] [Q&A]
Dear Developer,

Thank you for addressing my issue. I am currently using the Juggluco WearOS version and have encountered a sensor malfunction, with the watch indicating "stopped working state=7". When attempting to connect the sensor with my phone, it prompts "Replace Sensor (365) Your Sensor is not working. Please remove your Sensor and start a new one." I have only been using this sensor for five days, and during this period, I've only used Juggluco and Juggluco WearOS. It shouldn't be a battery fault theoretically, as the normal usage period is 14 days. I used the 'turbo','history bluetoorh' option in Bluetooth settings, and when I found that the data wasn't successfully uploading to my server over Wi-Fi, I switched to cellular data transfer. I wouldn't expect this operation to cause sensor damage. 

I am using the libre3 European version.
The watch's operating system initially was Android 9, but the app would automatically exit after a while, so later on, I switched to using Android 8. Yes, I used NFC to re-pair the device about seven or eight times.

Has anyone else reported this issue? How should this problem be resolved?

**Comment by j-kaltes** [comment]
If you follow other forums about the use of Freestyle Libre 3 sensors, you will come to know that these sensors regularly malfunction. In that case you need to contact Abbott’s customer service.
I have never seen a case in which that was because of a sensor battery problem. Mostly the sensor also keeps sending data over Bluetooth, but there are no new measurements.
